src/main/kotlin/uk/gov/justice/digital/hmpps/approvedpremisesapi/service/BookingService.kt

```diff
@@ -238,7 +238,7 @@ class BookingService(
       val lengthOfStayWeeksWholeNumber = (lengthOfStayDays.toDouble() % 7) == 0.0
 
       emailNotificationService.sendEmail(
```

Comment:
I think we should have two calls to `emailNotificationService.sendEmail` here, as it makes the function more generic, and gives us the flexibility to use another template for the premises if we need it